index.html

```diff
@@ -20,7 +20,7 @@
             href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@500&family=Russo+One&display=swap"
             rel="stylesheet"
         />
-
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Font awesome is not consumed in the project right,  wont it be better if we remove the cdn just to make code look clean